specification/src/chapter2.adoc

```diff
@@ -2,9 +2,600 @@
+
+Attacks can be direct or indirect:
+
+* Direct +
+An adversary gains direct access to a resource belonging to the victim. For example: direct access to a memory location or execution state, or direct control of the control flow of a victim.
+
+* Indirect +
+An adversary can access or modify the content of a resource by a side channel. For example: by analyzing timing patterns of an operation by a victim to reveal information about data used in that operation, or launching row-hammer style memory attacks to affect the contents of memory owned by the victim.
+
+* Chained +
+An adversary is able to chain together multiple direct and indirect attacks to achieve a goal. For example, use a software interface exploit to affect a call stack, and use that to take redirect the control flow of a victim.
+
+This specification is primarily concerned with ISA level mitigations against logical attacks.
+
+Physical or remote attacks in general need to be addressed at system, protocol or governance level, and may require additional non-ISA mitigations. However, some ISA level mitigations can also help provide some mitigation against physical or remote attacks and this is indicated in the tables below.
+
+The required level of protection can vary depending on use case. For example, a HW RoT may have stronger requirements on physical resistance than other parts of an SoC.
+
+Finally, this specification does not attempt to rate attacks by severity, or by adversary skill level. Ratings tend to depend on use case specific threat models and requirements. 
+
+==== Logical
+
+[width=100%]
+[%header, cols="5,5,5,10,15,10"]
+|===
+| ID#
+| Attack   
+| Type  
+| Description
+| Risc-V mitigations 
+| Planned mitigations
+
+| CAT_NNN
+| Unrestricted access 
+| Direct +
+Logical
+| Direct access to unauthroized resources in normal operation.
+a| * Risc-V privilege levels
+* Risc-V isolation
+* Risc-V virtualization
+| 
+
+| CAT_NNN
+| Transient execution attacks
+| Chained +
+Logical
+| Attacks on speculative execution implementations. 
+| Most of these attacks except Spectre v1 are specific to particular micro architectures, and Risc-V systems are expected not to be vulnerable. +
+For example: +
+https://meltdownattack.com/[Spectre and meltdown papers] +
+ https://www.intel.com/content/www/us/en/developer/topic-technology/software-security-guidance/processors-affected-consolidated-product-cpu-model.html[Intel security guidance] +
+https://developer.arm.com/documentation/#cf-navigationhierarchiesproducts=Arm%20Security%20Center,Speculative%20Processor%20Vulnerability[Arm speculative vulnerability]
+| Fence.t could mitigate against Spectre v1.
+
+| CAT_NNN
+| Interface abuse
+| Chained +
+Logical
+| Abusing interfaces across privilege or isolation boundaries, for example to elevate privilege or to gain unauthorized access to resources.
+a| * Risc V privilege levels
+* Risc-V isolation
```

Comment:
```suggestion
* RISC-V Supervisor domains (MTT)
```

Reply:
This is a catch-all for any Risc-V isolation mechanism. These are enumerated later. It's not just supervisor domains, and not just MTT.

Do you agree?